fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -26,19 +27,21 @@
      * @param timeGrain  time grain of the table
      * @param columns  The columns for this table
      * @param logicalToPhysicalColumnNames  Mappings from logical to physical names
+     * @param metadataService datasource metadata service containing availability data for the table
      */
     public ConcretePhysicalTable(
             @NotNull TableName name,
             @NotNull ZonedTimeGrain timeGrain,
-            @NotNull Iterable<Column> columns,
-            @NotNull Map<String, String> logicalToPhysicalColumnNames
+            @NotNull Set<Column> columns,
```

Comment:
Why the change from an `Iterable` to a `Set`? I believe @michael-mclawhorn made it an iterable, precisely to try to get across the idea that "Iteration order doesn't really matter, but we'll preserve it, so don't worry if it does."

`Set` doesn't really get that across.